Services/Services/FileService.cs

```diff
@@ -0,0 +1,75 @@
+/*
+* Digital Excellence Copyright (C) 2020 Brend Smits
+* 
+* This program is free software: you can redistribute it and/or modify 
+* it under the terms of the GNU Lesser General Public License as published 
+* by the Free Software Foundation version 3 of the License.
+* 
+* This program is distributed in the hope that it will be useful, 
+* but WITHOUT ANY WARRANTY; without even the implied warranty 
+* of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. 
+* See the GNU Lesser General Public License for more details.
+* 
+* You can find a copy of the GNU Lesser General Public License 
+* along with this program, in the LICENSE.md file in the root project directory.
+* If not, see https://www.gnu.org/licenses/lgpl-3.0.txt
+*/
+
+using Models;
+using Repositories;
+using Services.Base;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace Services.Services
+{
+
+    public interface IFileService : IService<File>
+    {
+
+        Task<List<File>> GetFileByFileIdAsync(int fileId);
+        Task<List<File>> GetFilesAsync();
+
+        void UploadSingleFile(File entity);
+
+    }
+
+    public class FileService : Service<File>, IFileService
+    {
+        public FileService(IFileRepository repository) : base(repository) { }
+
+        protected new IFileRepository Repository => (IFileRepository) base.Repository;
+
+        public async Task<List<File>> GetFileByFileIdAsync(int fileId)
+        {
+            return await Repository.GetFileByFileIdAsync(fileId).ConfigureAwait(false);
+        }
+
+        public async Task<List<File>> GetFilesAsync()
+        {
+            return await Repository.GetFilesAsync();
+        }
+
+        public override void Add(File entity)
+        {
+            base.Add(entity);
+        }
+
+        public override void Update(File entity)
+        {
+            base.Update(entity);
+        }
+
+        public void UploadSingleFile(File entity)
+        {
+            entity.UploadDateTime = DateTime.Now;
```

Comment:
Need to make the filename unique either here or in fileuploader but in my opinion it fits best in the service, I think this will prevent a lot of future headaches.

Reply:
Not yet done, should I change model and make path unique?

Reply:
yea sounds good, on another note maybe some checking for illegal characters in the name and replace them with dashes. In another project we had to replace slashes etc since it screwed with the filesystem.

Reply:
> looks good, checking and making name unique still needs to be implemented. When it's done I'll run the postman tests again. Btw on the file&variable for postman i found a request for implementation but also a workaround: [postmanlabs/postman-app-support#3641 (comment)](https://github.com/postmanlabs/postman-app-support/issues/3641#issuecomment-690281504)

I used a very similar method. Unfortunately not working in Postman application for me.

Reply:
Alright too bad, need to wait until it's implemented then